src/inputs/SelectField.test.tsx

```diff
@@ -148,6 +148,36 @@ describe("SelectFieldTest", () => {
     expect(onSelect).not.toHaveBeenCalled();
   });
 
+  it("can disable options with tooltips", async () => {
+    const onSelect = jest.fn();
+    // Given a Select Field with a disabled option
+    const { age, getByRole } = await render(
+      <SelectField
+        label="Age"
+        value="1"
+        options={options}
+        getOptionLabel={(o) => o.name}
+        getOptionValue={(o) => o.id}
+        data-testid="age"
+        disabledOptions={[["2", "Example Tooltip"]]}
```

Comment:
Thought: Wdyt about a hash like:

```
disabledOptions={[{ value: "2", reason: "Example Tooltip" }]}
```

?

Reply:
yeah that's much cleaner - I'll make that change!